### DISCUSSION TITLE
fundamentalRegionEuler() use

### DISCUSSION THREAD

**Discussion by RammyCa** [mtex-toolbox/mtex#1914] [Ask Anything]
Hi,

I want to change my euler angle ranges outside of my current CS and SS, but without declaring a different unit cell, and then compute an ODF with said abritrary ranges. How can I do this? The fundamentalRegionEuler() page is not very clear on its use.

**Comment by ralfHielscher** [comment]
What kind of data do you have?

RammyCa ***@***.***> schrieb am Di., 1. Aug. 2023, 16:01:

**Reply by RammyCa** [reply]
I have csv files with pole figures that I transform to an ODF. My crystal geometry is hexagonal close packed, so normally my limits are set to 360 90 60 but I want to augment it.

**Reply by ralfHielscher** [reply]
The second question would be what you want to do with the ODF at the larger limits. Actually, it is already "computed" for "all" Euler angles. Just do
```matlab
ori =orietation.byEuler(300*degree,170*degree, 350*degree,cs)
odf.eval(ori)
```
and you will get the value of the ODF at this specific orientation. I guess however, you think about plotting. In this case
```matlab
plot(odf,'complete')
```
will give you the desired result.
Ralf.
